stage1/passwd/group

```diff
@@ -0,0 +1,3 @@
+root:x:0:
+messagebus:x:106:
+rkt:x:1000:1000
```

Comment:
should be probably removed for now
It was me playing with running non-root containers, so it's not necessary